protos/local/workflow_completion.proto

```diff
@@ -7,8 +7,8 @@ import "workflow_commands.proto";
 
 /// Result of a single workflow activation, reported from lang to core
 message WFActivationCompletion {
-    // The token from the workflow activation you are completing
-    bytes task_token = 1;
+    // The run id from the workflow activation you are completing
+    string run_id = 1;
```

Comment:
We should probably add a unique identifier per activation for debug purposes.

Reply:
Let's save this for tracing/logging interface changes as it probably overlaps a lot.